deploy/hypershift-cluster-monitoring-config/cluster-monitoring-config.yaml

```
@@ -0,0 +1,118 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hshift-cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    enableUserWorkload: true
+    prometheusK8s:
+    # If the location of this config changes,
+    # make sure to adjust https://github.com/openshift/ops-sop/blob/master/v4/howto/observability/o11m_mst_killswitch.md
+    # so that killswitch instructions point to the right file!
+      remoteWrite:
+            - url: https://observatorium-mst.api.stage.openshift.com/api/metrics/v1/hypershift-platform/api/v1/receive
+      oauth2:
+        clientId:
+          secret:
+            key: client-id
+            name: rhobs-observatorium-credentials
```

This is the correct secret name deployed in the management clusters.
```suggestion
            name: rhobs-hypershift-credential
```